flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontend.java

```diff
@@ -727,10 +725,14 @@ private void disposeSavepoint(ClusterClient<?> clusterClient, String savepointPa
 	//  Interaction with programs and JobManager
 	// --------------------------------------------------------------------------------------------
 
-	protected void executeProgram(PackagedProgram program, ClusterClient<?> client, int parallelism) throws ProgramMissingJobException, ProgramInvocationException {
+	protected void executeProgram(
+		PackagedProgram program,
```

Comment:
Please add one more tab. Although not enforced, the arguments should be one tab more to the inside compared to the body of the method.